### PR TITLE
Aligning and updating RHEL 8 STIG w/ V1R2

### PR DESCRIPTION
**Pull request by carlosmmatos**
#### Description:

Aligning our RHEL 8 STIG profiles with DISA's latest release of V1R2.

### CODE REVIEW

File: rhel8/profiles/stig.profile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 documentation_complete: true
 
 metadata:
-    version: V1R1
+    version: V1R@
```
Comment (ggbecker) [inline rhel8/profiles/stig.profile:4]:
```suggestion
    version: V1R2
```

Comment (carlosmmatos) [reply rhel8/profiles/stig.profile]: dope!! okay it should be fixed now.

**Review by ggbecker — APPROVED** [review]
LGTM

**Pull request merged**